bundles/org.openhab.binding.meross/src/main/resources/OH-INF/thing/msg100.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="meross"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="msg100">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="gateway"/>
+		</supported-bridge-type-refs>
+
+		<label>Meross MSG100</label>
+		<description>Meross Garage Door</description>
+
+		<channels>
+			<channel id="doorState" typeId="doorState"/>
+		</channels>
+
+		<config-description>
+			<parameter name="name" type="text" required="true">
+				<label>Name</label>
+				<description>The name of the device as registered to Meross account</description>
+			</parameter>
+			<parameter name="uuid" type="text" required="true">
+				<label>Device UUID</label>
+				<description>Meross UUID of the device, found during discovery</description>
+			</parameter>
+			<parameter name="ipAddress" type="text">
```

Comment:
I think hostname is more appropiate.

Reply:
According to the description it can be set but is not required. When set it is used for local communication, host name should work. 
Anyway I’m fine leaving it as per your comment.